[DOC] Fix issues in consumer offset synchronization docs (#5607)
strimzi/strimzi-kafka-operator: documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

Offset synchronization is particularly useful in an _active/passive_ configuration.
If the active cluster goes down, consumer applications can switch to the passive (standby) cluster and pick up from the last transferred offset position.

To use topic offset synchronization, enable the synchronization by adding `sync.group.offsets.enabled` to the checkpoint connector configuration, and setting the property to `true`.
Synchronization is disabled by default.

> When using the `IdentityReplicationPolicy` in the source connector, it has to be configured also in the checkpoint connector configuration.

Reviewer: ```suggestion
When using the `IdentityReplicationPolicy` in the source connector, it also has to be configured in the checkpoint connector configuration.
```

> That will ensure that the mirrored consumer offsets will be applied for the correct topics.

Reviewer: ```suggestion
This ensures that the mirrored consumer offsets will be applied for the correct topics.
```

> The consumer offsets will be synchronized only for consumer groups which are not active in the target cluster.

Reviewer: ```suggestion
The consumer offsets are only synchronized for consumer groups that are not active in the target cluster.
```
